components/rmrk/Gallery/Gallery.vue

```diff
@@ -169,7 +160,7 @@ export default class Gallery extends mixins(
   }
 
   get isRmrk(): boolean {
-    return this.urlPrefix === 'rmrk' || this.urlPrefix === 'westend'
+    return false
   }
```

Comment:
I would like to remove `isRmrk` function and also `this.isRmrk` in this file

Reply:
Thanks for the comment! should be resolved